pkgs/by-name/we/webrtc-audio-processing/package.nix

```
@@ -0,0 +1,53 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitLab,
+  abseil-cpp,
+  meson,
+  ninja,
+  pkg-config,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "webrtc-audio-processing";
+  version = "2.1";
+
+  src = fetchFromGitLab {
+    domain = "gitlab.freedesktop.org";
+    owner = "pulseaudio";
+    repo = "webrtc-audio-processing";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-YR4ELukJgHMbfe80H+r8OiaZUCAqefGXmVOaTVVgOqA=";
+  };
+
+  outputs = [
+    "out"
+    "dev"
+  ];
+
+  nativeBuildInputs = [
+    meson
+    ninja
+    pkg-config
+  ];
+
+  propagatedBuildInputs = [
+    abseil-cpp
+  ];
+
+  mesonFlags = lib.lists.optional (!stdenv.hostPlatform.isAarch64) "-Dneon=disabled";
+
+  meta = {
+    homepage = "https://www.freedesktop.org/software/pulseaudio/webrtc-audio-processing";
+    description = "More Linux packaging friendly copy of the AudioProcessing module from the WebRTC project";
+    license = lib.licenses.bsd3;
+    maintainers = with lib.maintainers; [ fzdslr ];
```

Sorry, should have put this before.

Should probably add the pkgconfig stuff just for smoke-testing

```suggestion
  passthru.tests.pkg-config = testers.hasPkgsConfigModules {
    packages = finalAttrs.finalPackage;
    versionCheck = true;
  };
    
  meta = {
    homepage = "https://www.freedesktop.org/software/pulseaudio/webrtc-audio-processing";
    description = "More Linux packaging friendly copy of the AudioProcessing module from the WebRTC project";
    pkgConfigModules = [ "webrtc-audio-processing" ];
    license = lib.licenses.bsd3;
    maintainers = with lib.maintainers; [ fzdslr ];
```